Content.Shared/Singularity/Components/ContainmentFieldGeneratorComponent.cs

```diff
@@ -69,6 +69,14 @@ public int PowerBuffer
     [DataField("idTag", customTypeSerializer: typeof(PrototypeIdSerializer<TagPrototype>))]
     public string IDTag = "EmitterBolt";
 
+    /// <summary>
+    /// Which fixture ID should test collision with from the entity that powers the generator?
+    /// Prevents the generator from being powered by fly-by fixtures.
+    /// </summary>
+    [ViewVariables(VVAccess.ReadWrite)]
+    [DataField]
```

Comment:
Remove the VV as its applied automatically.

Reply:
Fixed!